blueocean-pipeline-api/src/main/java/io/jenkins/blueocean/pipeline/creation/AbstractMultiBranchCreateRequest.java

```diff
@@ -0,0 +1,132 @@
+package io.jenkins.blueocean.pipeline.creation;
```

Comment:
Hmm,  I am going to check. We are using credentials apis for everything, why are our classes getting in there? I will check sometime tonight or tomorrow morning.